Meadowlark-js/.env.example

```
@@ -2,3 +2,7 @@
 
 POSTGRES_USER=<postgres user>
 POSTGRES_PASSWORD=<postgres password>
+
+# OAuth 2.0 signing key: base 64 encoded, 256 bit key. To create a new one,
+# try running the application and accessing endpoint /{stage}/createSigningKey.
+SIGNING_KEY=v/AbsYGRvIfCf1bxufA6+Ras5NR+kIroLUg5RKYMjmqvNa1fVanmPBXKFH+MD1TPHpSgna0g+6oRnmRGUme6vJ7x91OA7Lp1hWzr6NnpdLYA9BmDHWjkRFvlx9bVmP+GTave2E4RAYa5b/qlvXOVnwaqEWzHxefqzkd1F1mQ6dVNFWYdiOmgw8ofQ87Xi1W0DkToRNS/Roc4rxby/BZwHUj7Y4tYdMpkWDMrZK6Vwat1KuPyiqsaBQYa9Xd0pxKqUOrAp8a+BFwiPfxf4nyVdOSAd77A/wuKIJaERNY5xJXUHwNgEOMf+Lg4032u4PnsnH7aJb2F4z8AhHldM6w5jw==
```

To keep people from blindly using this key, let's put a truly fake value in there.
```suggestion
SIGNING_KEY=<signing key>
```